src-java/kilda-pce/src/main/java/org/openkilda/pce/PathComputer.java

```diff
@@ -80,6 +80,15 @@ default GetHaPathsResult getHaPath(HaFlow haFlow, boolean isProtected)
     GetHaPathsResult getHaPath(HaFlow haFlow, Collection<PathId> reuseSubPathsResources, boolean isProtected)
             throws UnroutableFlowException, RecoverableException;
 
+    /**
+     * Calculates a protected path based on given parameters and configuration.
```

Comment:
should we use 'the' here?
```suggestion
     * Calculates a protected path based on the given parameters and configuration.
```